Gems/Multiplayer/Code/Source/Debug/MultiplayerConnectionViewportMessageSystemComponent.cpp

```diff
@@ -112,61 +112,75 @@ namespace Multiplayer
         if (agentType == MultiplayerAgentType::Client)
         {        
             // Display the connection status in the bottom-right viewport
+            m_drawParams.m_hAlign = AzFramework::TextHorizontalAlignment::Right;
+            m_drawParams.m_position = AZ::Vector3(static_cast<float>(viewportSize.m_width), static_cast<float>(viewportSize.m_height), 1.0f) + AZ::Vector3(viewportConnectionBottomRightBorderPadding) * viewport->GetDpiScalingFactor();
```

Comment:
If width and height are uints here, prefer aznumeric_cast.

Reply:
yes! fixed